### PR TITLE
new test notify

### PR DESCRIPTION
**Pull request by chenyan19209219**

### CODE REVIEW

**Comment by qiqi19209219** [comment]
LGTM

**Comment by chenyan19209219** [comment]
> LGTM

reply2

File: src/main/1.patch [hunk 1]
```diff
@@ -0,0 +1,18 @@
+diff --git a/1.txt b/1.txt
+new file mode 100644
+index 0000000000000000000000000000000000000000..706c11b16b808e6219cf6ef6bffad93e381a0a68
+--- /dev/null
++++ b/1.txt
+@@ -0,0 +1,2 @@
++first add on develop branch.
++this is line 2
+\ No newline at end of file
```
Comment (katerina19209219) [inline src/main/1.patch]: 评1

File: src/main/1.patch [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+diff --git a/2.txt b/2.txt
+new file mode 100644
+index 0000000000000000000000000000000000000000..08f1b0cb2865653b7ae76b2e7ff59e997d88df2d
+--- /dev/null
```
Comment (katerina19209219) [inline src/main/1.patch]: 评2

File: src/main/java/math/Sigmoid.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        return result;
+    }
+
+    public static double sigmoidWithWeight(double value, double weight) {
+        double ey = Math.pow(Math.E, - weight * value);
+        double result = 1 / (1 + ey);
+        return result;
+    }
+
+    public static void main(String[] args) {
```
Comment (katerina19209219) [inline src/main/java/math/Sigmoid.java]: 1

File: src/main/java/math/Sigmoid.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        double sigmoid = sigmoid(10);
+        System.out.println(sigmoid);
```
Comment (katerina19209219) [inline src/main/java/math/Sigmoid.java]: 2

**Comment by chenyan19209219** [comment]
123123

**Comment by katerina19209219** [comment]
467b159abaa67973a94a305e81cc633b

upload a file

**Comment by katerina19209219** [comment]
> 467b159abaa67973a94a305e81cc633b
> 
> upload a file

123123123

File: pom.xml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns:xsi="http://www.w3.org/2001/XMLaachema-instance"
```
[the remark this answers is not in the document]
Comment (katerina19209219) [reply pom.xml]: 123123

File: src/main/1.patch (hunk 1 above)
Comment (katerina19209219) [reply src/main/1.patch]: > > 467b159abaa67973a94a305e81cc633b
> > upload a file
> 
> 123123123

从VB你

**Comment by chenyan19209219** [comment]
**Test**

**Pull request closed without merging**